src/main/java/io/jenkins/plugins/artifact_manager_s3/JCloudsArtifactManager.java

```diff
@@ -83,6 +83,8 @@
 
     private static String BLOB_CONTAINER = System.getenv("S3_BUCKET");
     private static String PREFIX = System.getenv("S3_DIR");
+    public static String AWS_REGION = System.getProperty(JCloudsArtifactManager.class.getName() + ".AWS_REGION");
```

Comment:
`getenv` for consistency I suppose.

Reply:
environment variables on unix can not have dots, if we want to use an environment variable we would have to use something plainer like S3_REGION or use AWS_REGION but I do not like to reuse an AWS environment variable.

Reply:
> I do not like to reuse an AWS environment variable

Why not?

Help me understand here. According to the [SDK docs](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#automatically-determine-the-aws-region-from-the-environment), merely setting the `AWS_REGION` environment variable would suffice to configure the region; as would defining it in `~/.aws/config`. If true, that means this PR is unnecessary and it is merely a matter of documentation that the admin is expected to properly configure their AWS environment on the Jenkins master node.

Reply:
there is another issue to move from env vars to system properties, so this is ok. IIRC in other places camel case is used IIRC `awsRegion`

Reply:
In some cases seems you have to force the region used to create the URLs, for example, if you are login in one Region and the bucket it is in another. This setting forces the Region to use with JClouds, Should it be a property or an environment variable?

Reply:
it should be a property because you will be submitting another PR to move from env vars to system properties correct ?

I would call it just region because it is in the not S3 specific class